js/src/textures/TextTexture.js

```diff
@@ -0,0 +1,52 @@
+var _ = require('underscore');
+var Promise = require('bluebird');
+var TextTextureBase = require('./TextTexture.autogen');
+
+var TextTextureView = TextTextureBase.TextTextureView.extend({});
+
+var TextTextureModel = TextTextureBase.TextTextureModel.extend({
```

Comment:
Why is this included in the PR? was this something the previous pythreejs depended on? This somewhat breaks from the auto-gen philosophy of *just* mirroring the api of three.js, but I'll defer to @jasongrout on this one.